Cargo.toml

```diff
@@ -13,50 +13,45 @@ repository = "https://github.com/stackabletech/secret-operator"
 [workspace.dependencies]
 anyhow = "1.0"
 async-trait = "0.1"
-bindgen = "0.66"
-built = { version = "0.6", features = ["chrono", "git2"] }
-byteorder = "1.4"
-clap = "4.3"
+bindgen = "0.69.4"
```

Comment:
major.minor please :)

Reply:
All done in 614fe210ae998ee134d0d62f237621370db050cc